### ISSUE TITLE
Allow jira link to be constructed without a Jira server integration

### ISSUE THREAD

**Issue by bethesque** [tomasbjerre/git-changelog-lib#68]
Due to some security restraints, we're not able to use a jira account to generate our changelog. I'd like to include a link to the Jira issue in the changelog, however. At the moment, I'm manually creating the link in the mustache template as so:

```
   {{^hasLink}}
### {{name}} [{{issue}}](https://jira.foo/browse/{{issue}}) {{title}} {{#hasIssueType}} *{{issueType}}* {{/hasIssueType}} {{#hasLabels}} {{#labels}} *{{.}}* {{/labels}} {{/hasLabels}}
   {{/hasLink}}
```

I'd like to be able to identify which issues are the Jira issues, so that I don't put in a link for issues that come from other systems. My initial thought was to expose an `isJira` property on the issue. What are your thoughts? I'm happy to do a PR for this.

**Comment by tomasbjerre** [comment]
Yes I would happliy accept such a PR. Perhaps it can add:

 * `isGitHub`
 * `isGitLab`
 * `isJira`
 * `isCustom`

You can probably also solve it by defining a custom issue. If you are using the Gradle plugin it would be somehting like this:
```
 customIssues = [
  [ "Jira", "\\b[a-zA-Z]([a-zA-Z]+)-([0-9]+)\\b", "https://jira.foo/browse/\${PATTERN_GROUP}", "\${PATTERN_GROUP}" ]
 ]
```

**Comment by tomasbjerre** [comment]
Yes. I also came to that conclusion. The code needs refactoring...

**Comment by bethesque** [comment]
Works like a shop bought one. Thanks!
